cypress/integration/pages/index.js

```diff
@@ -96,6 +96,19 @@ const runCommonTests = ({ service, pageType }) => {
         );
       });
 
+      if (Cypress.env('APP_ENV') === 'live') {
+        it('footer links should not 404', () => {
+          cy.get('a')
+            .not('[href="#content"]')
```

Comment:
This seems a little too specific, but a .not (starting with #) would be ideal I think.